tests/conftest.py

```diff
@@ -200,7 +225,6 @@ def dummy_carbon_data(layer_roles_fixture):
             "cell_id": data.grid.cell_id,
         }
     )
-    # TODO - Eventually this should replace the dummy soil moisture entirely
     data["matric_potential"] = xr.concat(
```

Comment:
Maybe we can chat about this, if you don't need soil moisture we can focus on the matric potential because this is also how P-Hydro uses

Reply:
Just made #345 to reflect our in person discussion of this